Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1498,7 +1498,7 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <!-- XHED: Explanation screen confirmed new infections title -->
     <string name="statistics_explanation_confirmed_new_infection_title">"Bestätigte Neuinfektionen"</string>
     <!-- XTXT: Explanation screen confirmed new infections text -->
-    <string name="statistics_explanation_confirmed_new_infection_text" formatted="false">"Anzahl der an das RKI übermittelten Corona-positiv getesteten Personen. Anzahlen unter einem Schwellenwert von 1% des 7-Tage-Mittelwerts werden nicht angezeigt; siehe auch Erläuterung unter Legende > Zeitraum."</string>
+    <string name="statistics_explanation_confirmed_new_infection_text">"Gesamtzahl der bestätigten Neuinfektionen der letzten 7 Tage (nach Meldedatum) pro 100.000 Einwohner."</string>
```

Comment:
Probably this will now display correctly on the screen because the strings have been exchanged, but the string names do not fit the content.
Here the string name is "statistics_explanation_new_infection_text", and the text is about new infections of the last 7days per 100.000 inhabitants. So this text is related to the 7day incidence and should rather be inserted for "statistics_explanation_seven_day_incidence_text".  And it must display under the title "7-Tage Inzidenz" (statistics_explanation_seven_day_incidence_title").
And for  "statistics_explanation_new_infection_text", the text should be "Anzahl der an das RKI übermittelten Corona-positiv getesteten Personen. Anzahlen unter einem Schwellenwert..."  and needs to be displayed under the title: "Bestätigte Neuinfektionen" (statistics_explanation_confirmed_new_infection_title)

Reply:
Ok, I have reverted the changes to the actual strings and just switched their use in the UI.